app/uk/gov/hmrc/eeittadminfrontend/models/DMSReconciliation.scala

```diff
@@ -26,18 +26,22 @@ case class DmsReport(data: List[DmsReportData], count: Int)
 case class DmsReportData(submissionRef: SubmissionRef, envelopeId: EnvelopeId)
 
 object DmsReport {
-  def apply(file: File): DmsReport = {
-    val source = Source.fromFile(file)
-    val lines = source.getLines().toList
-    source.close()
+  def apply(file: File): DmsReport =
+    try {
+      val source = Source.fromFile(file)
+      val lines = source.getLines().toList
+      source.close()
```

Comment:
Can we use the `Using` utility from the scala.util package or put it into finally block?